plugins/org.yakindu.sct.doc.user/src/user-guide/sctunit.textile

```diff
@@ -957,6 +976,6 @@ p(#sctunit_fig_test_unit_grammar).
 
 !images/sctunit_grammar_030_test_unit.png(Test unit grammar)!
 
-p=. Test unit grammar
+p=. Test unit grammar %{display: block; font-size: 50%; padding-left: 20%; padding-right: 20%;}*Please note:* The _PackageImport_ non-terminal symbol will be available in future versions of YAKINDU Statechart Tools.%
```

Comment:
Therefore it would be better if we use an updated graphic without the PackageImport